server/src/main/java/org/elasticsearch/index/snapshots/blobstore/SnapshotFiles.java

```diff
@@ -128,4 +128,8 @@ private FileInfo findPhysicalIndexFile(String physicalName) {
         return physicalFiles.get(physicalName);
     }
 
+    @Override
+    public String toString() {
+        return "SnapshotFiles{" + indexFiles.toString() + "}";
```

Comment:
Can we add `shardStateIdentifier` here while we're at it ? :) (also `.toString()` isn't necessary?

Reply:
yep, see https://github.com/elastic/elasticsearch/pull/66162/commits/63383c77547a0760a274d83ab026dd62eb6fffc9